File: orm/src/test/java/com/example/orm/PersonResourceTest.java
package com.example.orm;

import io.quarkus.test.junit.QuarkusTest;
import org.junit.jupiter.api.Test;

import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.is;

@QuarkusTest
public class PersonResourceTest {

    @Test
    public void testHelloEndpoint() {
        given()
          .when().get("/person")
          .then()
             .statusCode(200)
             .body(is("hello"));
    }

}

File: kotlin-orm/src/test/kotlin/com/example/orm/PersonResourceTest.kt
package com.example.orm

import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured.given
import org.hamcrest.CoreMatchers.`is`
import org.junit.jupiter.api.Test

@QuarkusTest
open class PersonResourceTest {

    @Test
    fun testHelloEndpoint() {
        given()
          .`when`().get("/person")
          .then()
             .statusCode(200)
             .body(`is`("hello"))
    }

}